### PR TITLE
Tweak WorkspaceConfig._origin docs

### PR DESCRIPTION
**Pull request by SamB**
Apparently vscode finds those newlines very tasty,
so use markdown to actually mark this up as a list.

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@SamB: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankeromnes** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankeromnes` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by gtsiolis — COMMENTED** [review]
Thanks for contributing, @SamB! 🌟

@jankeromnes could you take a look at this? Changes seem trivial but wanted to loop you in for another pair of eyes. 🏓

@SamB You'll also need to sign a Contributor License Agreement (CLA)[1] once before merging your first contribution. Looping in @meysholdt to reach out about the CLA. 🏓

**Comment by SamB** [comment]
BTW, https://www.gitpod.io/cla redirects to https://github.com/gitpod-io/cla but that's not a thing, see https://github.com/gitpod-io/website/blob/ed6ee0e2958c494b850673fd2eb087d9ce060e2e/_redirects#L4

File: components/gitpod-protocol/src/protocol.ts
Comment (jankeromnes) [inline components/gitpod-protocol/src/protocol.ts:572]: Could also have a `:` to avoid the double `-`:

```suggestion
     * - repo: from the repository
     * - project-db: from the "Project" stored in the database
     * - definitly-gp: from github.com/gitpod-io/definitely-gp
     * - derived: computed based on analyzing the repository
     * - additional-content: config comes from additional content, usually provided through the project's configuration
     * - default: our static catch-all default config
```

**Review by jankeromnes — APPROVED** [review]
Looks good to me, thanks! 🙏

> BTW, https://www.gitpod.io/cla redirects to https://github.com/gitpod-io/cla but that's not a thing, see https://github.com/gitpod-io/website/blob/ed6ee0e2958c494b850673fd2eb087d9ce060e2e/_redirects#L4

Whoops, great catch! 🔍 We should fix that -- reported to the team.

---

Approving (with one suggestion), but keeping the hold until the CLA is confirmed.

/lgtm
/approve no-issue
/hold

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: dcfb362947fd5b4d1e7cf41780525a2cf8f26063</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7069#pullrequestreview-826252511" title="Approved">jankeromnes</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7069#pullrequestreview-826252511" title="Approved">jankeromnes</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by meysholdt** [comment]
hi Samuel,

I've sent you a CLA via DocuSign. Can you please sign it so that we can merge this PR?
thank you!

**Comment by meysholdt** [comment]
Hi Samuel,

We did not yet receive a signature on the CLA from you. I've re-sent the request via DocuSign to you. 
If you have questions or do not feel comfortable signing, please drop me a line via `moritz@gitpod.io`.

Regards,
  Moritz

**Comment by jankeromnes** [comment]
Friendly ping @SamB -- also, a very happy New Year 2022! 🎆 🙌

**Pull request closed without merging**
